tb/core/firmware/start.S

```diff
@@ -317,17 +317,6 @@ start:
 
 	TEST(fence_i)
 
-	TEST(amoadd_w)
-	TEST(amoand_w)
-	TEST(amomaxu_w)
-	TEST(amomax_w)
-	TEST(amominu_w)
-	TEST(amomin_w)
-	TEST(amoor_w)
-	TEST(amoswap_w)
-	TEST(amoxor_w)
-	TEST(lrsc)
-
```

Comment:
Why is this removed? These test should work.

Reply:
The RVA parameter has been set to 0 by default, so they generate illegal instructions now